Organiser.Core/Organiser.Core/Context/DataBaseContext.cs

```diff
@@ -87,6 +87,18 @@ public void CreateOrUpdate(Savings saving)
         public IQueryable<Savings> AllSavings => dataContext.Savings;
         #endregion
 
+        #region Bugs
+        public IQueryable<Bugs> Bugs => dataContext.Bugs.Where(x => x.BUID == user.UID);
+        public IQueryable<Bugs> AllBugs => dataContext.Bugs;
+        public void CreateOrUpdate(Bugs bug)
```

Comment:
'CreateOrUpdate' suggests that you allow for two operations in single method, which a code smell